tests/integration/query_range_test.go

```diff
@@ -0,0 +1,176 @@
+//go:build integration && go1.22 && goexperiment.rangefunc
+// +build integration,go1.22,goexperiment.rangefunc
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ydb-platform/ydb-go-sdk/v3"
+	"github.com/ydb-platform/ydb-go-sdk/v3/internal/version"
+	"github.com/ydb-platform/ydb-go-sdk/v3/internal/xtest"
+	"github.com/ydb-platform/ydb-go-sdk/v3/query"
+)
+
+func TestQueryRange(t *testing.T) {
+	if version.Lt(os.Getenv("YDB_VERSION"), "24.1") {
+		t.Skip("query service not allowed in YDB version '" + os.Getenv("YDB_VERSION") + "'")
+	}
+
+	ctx, cancel := context.WithCancel(xtest.Context(t))
+	defer cancel()
+
+	db, err := ydb.Open(ctx,
+		os.Getenv("YDB_CONNECTION_STRING"),
+		ydb.WithAccessTokenCredentials(os.Getenv("YDB_ACCESS_TOKEN_CREDENTIALS")),
+	)
+	require.NoError(t, err)
+	t.Run("Execute", func(t *testing.T) {
+		var (
+			p1 string
+			p2 uint64
+			p3 time.Duration
+		)
+		r, err := db.Query().Execute(ctx, `
```

Comment:
Need test with more, then one line and more, then one result set - for check iterations.